test/router-options.test.js

```diff
@@ -58,3 +58,20 @@ test('Should honor maxParamLength option', t => {
     t.strictEqual(res.statusCode, 404)
   })
 })
+
+test('Should handler in shorthand route options take effect if there is no handler function as the third parameter and the second parameter is an object', t => {
```

Comment:
@shuperry This test is actually in the wrong file. It should be in `test/route.test.js`.

Reply:
@nwoltman OK, I will put the test in `test/route.test.js`.

Reply:
@nwoltman 
1. I'm thinking this test is already in the right file, this test is used for testing the options in shorthand router, the tests above this test has the same logic.

2. This file `test/route.test.js` does not allow shorthand route, even I only write the code below, there will be an error said `not ok root plugin has already booted`.

![a521fae7-99c5-4804-8495-98750985dbcf](https://user-images.githubusercontent.com/12183336/36954200-5ff6ab70-205b-11e8-9964-46d01998dcfd.png)

Reply:
1. The tests in the file are actually just for testing the options that get passed to `find-my-way` ([see here](https://github.com/fastify/fastify/blob/288903a3504c98762005e77c5b100ed272e0d352/fastify.js#L66-L70)). Testing the `fastify.route()` method and the shorthand methods should happen in `test/route.test.js`.
2. Sorry about the `not ok root plugin has already booted` error. There are some problems with the `test/route.test.js` file right now. I'm going to try to fix that file so that the error won't happen.

Reply:
@shuperry The `root plugin has already booted` error should be fixed now if you fetch and rebase the latest changes from master.

Reply:
@nwoltman I already moved the test to `test/route.test.js`.